dvc/command/plots.py

```diff
@@ -133,10 +133,7 @@ def add_parser(subparsers, parent_parser):
     _add_output_arguments(plots_show_parser)
     plots_show_parser.set_defaults(func=CmdPlotsShow)
 
-    PLOTS_DIFF_HELP = (
-        "Plot differences in metrics between commits in the DVC "
-        "repository, or between the last commit and the workspace."
-    )
+    PLOTS_DIFF_HELP = "Plot differences in metrics between commits."
```

Comment:
I think phrasing here (in original `DIFF_HELP` too) is a bit off. We never plot differences. Plots exist alongside, either on the same plot (`default`, `scatter`) or on separate canvases (confusion matrix). 

In the case of metrics, we do `diff` them by subtracting. Word `diff` suggests visualizing some kind of difference between metrics, which is not happening in `plots`.
I am not sure whether we should treat the plot in a similar way we do with metrics.

Reply:
@pared but the command is called `diff` :) A few lines on the same plot could also be considered as diff, that's what we do and why the command is called `dvc plots diff`. As you've mentioned, we don't really plot all plots in the same image, but rather we show 1 image per 1 plot with multiple lines in it, and that is how we visualise a diff for a particular plot.